doc/syslog/syslog-design.md

```diff
@@ -444,7 +657,7 @@ config syslog add '2.2.2.2' \
 config syslog del '2.2.2.2'
 ```
 
-#### 2.6.2.2 Show command group
+#### 3.6.2.2. Show command group
 
 **The following command shows syslog server configuration:**
```

Comment:
The extended parameters shall be displayed in show command output also for consistency.

Reply:
added